src/packs.c

```diff
@@ -325,7 +327,13 @@ int download_subscribed_packs(struct list *subs, struct manifest *mom, bool requ
 		}
 	}
 	list_free_list(need_download);
+
+	progress_set_next_step("pack_extraction");
 	info("Finishing packs extraction...\n");
+	progress_report_indeterminate();
 
-	return swupd_curl_parallel_download_end(download_handle, NULL);
+	err = swupd_curl_parallel_download_end(download_handle, NULL);
+	progress_report(1, 1);
```

Comment:
progress_complete_step should already send a progress_report(1, 1)

Reply:
will fix